manual_user_install.sh

```diff
@@ -0,0 +1,235 @@
+
+    # install speech to text (stt)
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-microphone-plugin-alsa
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-server
+
+    # install text to speech (tts)
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic3-server
+    sudo apt install -y espeak-ng
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-piper
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-server-plugin
+
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-config
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-utils
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-bus-client
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-plugin-manager
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-workshop
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-lingua-franca
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-cli-client
+
+    # install phal components
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-connectivity-events
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-system
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL-plugin-ipgeo
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL-plugin-oauth
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-dashboard
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-alsa
+
+    # install required skills
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-volume
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-fallback-unknown
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-stop
+    pip3 install git+https://github.com/OpenVoiceOS/skill-alerts
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-personal
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-naptime
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-date-time
+
+    # You can uncomment these lines if the deprecation notices are flooding your logs or slowing the system
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/fingerprinting.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/configuration.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/ovos_service_api.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/signal.py
+
+    if [[ ! -d $HOME/.config/mycroft ]]; then
+        mkdir -p $HOME/.config/mycroft
+    fi
+    cp $HOME/raspbian-ovos/stage-ovos/02-ovos/files/mycroft.conf $HOME/.config/mycroft/
+
+    echo
+    echo "Done installing OVOS core"
+    }
+
+function install_systemd (){
+    echo
+    echo "Installing systemd files"
+    echo
+
+    cd $HOME/raspbian-ovos/stage-ovos/00-system/files
```

Comment:
```suggestion
    cd $SCRIPT_DIR/raspbian-ovos/stage-ovos/00-system/files
```

Reply:
Implemented and tested in my current working copy.